src/admin/content-block/helpers/generators/accordions.ts

```diff
@@ -24,6 +24,7 @@ export const INITIAL_ACCORDIONS_BLOCK_STATE = (position: number): DefaultContent
 
 export const ACCORDIONS_BLOCK_CONFIG = (position: number = 0): ContentBlockConfig => ({
 	name: i18n.t('admin/content-block/helpers/generators/accordions___accordeons'),
+	type: ContentBlockType.Accordions,
```

Comment:
Bloktype wordt volgens mij al bijgehouden in de `blockState` via de initial state constante.

![image](https://user-images.githubusercontent.com/36598928/74732444-8de6eb80-524a-11ea-9927-63fee305f16b.png)

Reply:
ok, fix ik in de toekomst